packages/ui/src/molecules/RadioGroup/RadioGroup.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { render } from '@testing-library/react'
+import { renderHook } from '@testing-library/react-hooks'
+import userEvent from '@testing-library/user-event'
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import { RadioGroup, RadioOption, useRadioOption } from '.'
+
+describe('RadioGroup', () => {
+  it('Should render radio group with radio option', () => {
+    const onChange = jest.fn()
+    const { getByTestId, getAllByLabelText } = render(
+      <RadioGroup name="radio-group" selectedValue="radio" onChange={onChange}>
+        <RadioOption value="radio" label="Radio 1" />
+      </RadioGroup>
+    )
+
+    const [radio] = getAllByLabelText(/Radio 1/i)
```

Comment:
I was looking for a way to query the radio more semantically via Testing Library and [found out](https://stackoverflow.com/a/66940477/7970105) we could use `getByRole('radio', { name: … })` here. 

```suggestion
    const { getByTestId, getByRole } = render(
      <RadioGroup name="radio-group" selectedValue="radio" onChange={onChange}>
        <RadioOption value="radio" label="Radio 1" />
      </RadioGroup>
    )

    const radio = getByRole('radio', { name: 'Radio 1' })
```

This asserts there's actually an element with the `radio` role!